### PR TITLE
fixing production issues found on staging

### PR DESCRIPTION
**Pull request by ToddKopriva**
There were some extraneous line breaks and bad tag in the staged documents. 

I also added links in the Container-Optimized OS documents for folks reading these documents on c.g.c., not GitHub. Without these links, the reader had no easy way of seeing the code.

### CODE REVIEW

File: tutorials/gcp-cos-basic-fim/index.md [hunk 1]
```diff
@@ -8,8 +8,9 @@ date_published: 2019-03-21
 
 ## Example simple file integrity monitoring (FIM) container
 
-A basic FIM Docker image with regularly scheduled scans.
-Designed to be run on Google Container-Optimized OS, but should work with most other Docker servers.
+[This example](https://github.com/GoogleCloudPlatform/community/tree/master/tutorials/gcp-cos-basic-fim) provides a basic FIM Docker image with regularly scheduled scans.
+
```
Comment (jennyrosewood) [inline tutorials/gcp-cos-basic-fim/index.md]: Is this blank line intentional?

File: tutorials/dotnet-mvms-quickstart.md [hunk 2]
```diff
@@ -201,7 +199,7 @@ Learn how to use App Engine with other Google Cloud Platform products:
 **Learn to use Cloud SQL.** Cloud SQL is a fully-managed, relational MySQL database for your applications. </walkthrough-tutorial-card><walkthrough-alt>Learn more in the [Cloud SQL documentation](https://cloud.google.com/appengine/docs/flexible/dotnet/using-cloud-sql).</walkthrough-alt>
 
 <walkthrough-tutorial-card url=https://cloud.google.com/appengine/docs/flexible/dotnet/using-cloud-storage icon="STORAGE_SECTION" label="cloudStorage">
-**Learn to use Cloud Storage** Cloud Storage is a powerful and simple object
+**Learn to use Cloud Storage.** Cloud Storage is a powerful and simple object
```
Comment (jennyrosewood) [inline tutorials/dotnet-mvms-quickstart.md]: Recommend avoiding language like powerful, simple, quick, easy, etc.

File: tutorials/gcp-cos-basic-fim/index.md (hunk 1 above)
Comment (ToddKopriva) [reply tutorials/gcp-cos-basic-fim/index.md]: Yep.

File: tutorials/dotnet-mvms-quickstart.md (hunk 2 above)
Comment (ToddKopriva) [reply tutorials/dotnet-mvms-quickstart.md]: Good point. I'll pass this along.

**Pull request merged**